changelog/README.md

```diff
@@ -2,6 +2,10 @@
 
 <!-- Note: contrary to 3.x, insert new entries *first* in their section -->
 
+### 4.0.0-rc1 (In progress)
+
+- [documentation] JAVA-2075: Document preference for LZ4 over Snappy
```

Comment:
I think in the past we usually didn't include documentation tasks in the changelog, but I'm OK with including them from now on.